src/components/ScrollButton.vue

```diff
@@ -2,20 +2,19 @@
   <button
     v-if="showBtn"
     :aria-label="$t('browse-page.aria.scroll')"
-    class="scroll"
+    class="scroll text-white bg-trans-blue hover:bg-trans-blue-action transition-all duration-100 ease-linear fixed right-4 bottom-4 w-14 h-14 hover:shadow-md rounded-full text-center"
```

Comment:
Can we use @apply here?

Reply:
I'll make a discussion today, but I think we need to talk about our Tailwind usage as a team before encouraging particular design patterns.